consent-ui/nginx.conf

```diff
@@ -0,0 +1,46 @@
+worker_processes auto;
+pid /run/nginx.pid;
+
+events {
+    worker_connections 1024;
+    multi_accept on;
+}
+
+http {
+    resolver 127.0.0.11 ipv6=off;
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 15;
+    types_hash_max_size 2048;
+    server_tokens off;
+
+    include /etc/nginx/mime.types;
+    default_type text/javascript;
+
+    access_log off;
+    error_log /var/log/nginx/error.log;
+
+    gzip on;
+    gzip_min_length 100;
+    gzip_http_version 1.1;
+    gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
+
+    client_max_body_size 8M;
+
+    server {
+      listen 4200;
+      root /app/dist;
+      add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
You need `"Access-Control-Expose-Headers": "*"` too
Also please create task to harden Consent-UI CORS

Reply:
done